src/main/scala/tech/cryptonomic/conseil/metadata/Path.scala

```diff
@@ -10,24 +10,29 @@ sealed trait Path {
 case class EmptyPath() extends Path {
   override def up: Path = this
   override def addLevel(platform: String): Path = PlatformPath(platform, this)
+  override def toString: String = ""
 }
 
 // case class representing a path for a platform
 case class PlatformPath(platform: String, up: EmptyPath = EmptyPath()) extends Path {
   override def addLevel(networkPath: String): NetworkPath = NetworkPath(networkPath, this)
+  override def toString: String = platform
 }
 
 // case class representing a path for a network
 case class NetworkPath(network: String, up: PlatformPath) extends Path {
   override def addLevel(entity: String): EntityPath = EntityPath(entity, this)
+  override def toString: String = s"${up.toString} / $network"
```

Comment:
No need to call `up.toString` explicitly, it's done by the string interpolator

Reply:
done